### PR TITLE
In Validation and Form Attributes

### PR DESCRIPTION
**Pull request by egoist-dd**
## Description

| Questions               | Answers |
|-------------------------|-------------------------------------------------------|
| Why?                    | Cast Page Value to valid Integer, to prevent wrong Data or Errors, Adding Form Attributes to View Data to Extend Forms with Attribute Fields |
| BC breaks?              |no |
| Tests exists & pass?    |no |
| Related tickets?        | - |
| How to test?            | - |
| Requirements met?       | - |

### CODE REVIEW

File: engine/Shopware/Controllers/Frontend/Forms.php [hunk 1]
```diff
@@ -196,6 +196,7 @@ protected function getContent($formId)
             'metaTitle'         => $form->getMetaTitle(),
             'metaDescription'   => $form->getMetaDescription(),
             'metaKeywords'      => $form->getMetaKeywords(),
+            'attribute'         => $form->getAttribute(),
```
Comment (mitelg) [inline engine/Shopware/Controllers/Frontend/Forms.php]: I think this is not necessary

**Review by mitelg — CHANGES_REQUESTED** [review]

**Comment by mitelg** [comment]
hey @egoist-dd 
thanks for your PR :+1: 

please resolve the conflicts and squash your commit to one

File: engine/Shopware/Controllers/Frontend/Forms.php (hunk 1 above)
Comment (egoist-dd) [reply engine/Shopware/Controllers/Frontend/Forms.php]: why? the systems allows me to add attributes to forms, the system reads attributes from database, but the attributes where not send to the view :-|

File: engine/Shopware/Controllers/Frontend/Forms.php [hunk 2]
```diff
@@ -217,6 +217,7 @@ protected function getContent($formId)
             'metaTitle' => $form->getMetaTitle(),
             'metaDescription' => $form->getMetaDescription(),
             'metaKeywords' => $form->getMetaKeywords(),
+            'attribute' => $form->getAttribute(),
```
Comment (mitelg) [inline engine/Shopware/Controllers/Frontend/Forms.php]: please provide this change in separate PR

**Review by mitelg — CHANGES_REQUESTED** [review]

**Pull request merged**